.github/workflows/command-compile.yml

```diff
@@ -0,0 +1,117 @@
+name: Compile Command
```

Comment:
This is not about compiling.
To not confuse it with the actual compiling command in all other repos can you please rename it and also not use `/compile` as command which is very misleading?

Reply:
Fine by me. However in server the compile command also compiles and commits changes of npm run sass and npm run icons IIRC. So it is not a big difference to what this is doing as far as I understand it...

Reply:
I've now used `/l10n update`. Is this good?